README.md

```diff
@@ -51,6 +51,7 @@ zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics. | [Linux](http
 
 Name     | Description | OS
 ---------|-------------|----
+bcache | Exposes bcache statistics | Linux
```

Comment:
Okay, updated. It would be interesting to know how collectors that are disabled by default are unreasonable, but the README hints at that information only for tcpstat.